core/security-keycloak/src/main/java/io/thorntail/security/keycloak/impl/KeycloakUtils.java

```diff
@@ -0,0 +1,61 @@
+package io.thorntail.security.keycloak.impl;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.keycloak.adapters.KeycloakDeployment;
+import org.keycloak.adapters.KeycloakDeploymentBuilder;
+import org.keycloak.representations.adapters.config.AdapterConfig;
+
+public final class KeycloakUtils {
+    private KeycloakUtils() {
+    }
+    
+    public static KeycloakDeployment loadFromClasspath(String path) {
+        if (path == null) {
+            return null;
+        }
+        if (path.startsWith("classpath:")) {
+            path = path.substring("classpath:".length());
+        }
+        URL url = KeycloakUtils.class.getClassLoader().getResource(path);
+        if (url == null) {
+            url = KeycloakUtils.class.getClassLoader().getResource("META-INF/" + path);
+        }
```

Comment:
I don't believe we need WEB-INF parsing as there is no "WAR" in v4. It's all just JARs